stereo_image_proc/launch/stereo_image_proc.launch.py

```diff
@@ -49,26 +76,55 @@ def generate_launch_description():
             name='use_system_default_qos', default_value='False',
             description='Use the RMW QoS settings for the image and camera info subscriptions.'
         ),
+        DeclareLaunchArgument(
+            name='left', default_value='left', description='Namespace for the left camera'
+        ),
+        DeclareLaunchArgument(
+            name='right', default_value='right', description='Namespace for the right camera'
```

Comment:
Would you mind making these a little more descriptive like `left_ns`?

Reply:
Sure. FWIW, I choose these names to match the [ROS 1 launch file](https://github.com/ros-perception/image_pipeline/blob/0a421c80d018fcc47aaaf96bb06a5e7c31734b7e/stereo_image_proc/launch/stereo_image_proc.launch#L6-L7).

Reply:
See 5aff814